api/src/snapshots/resolvers/veleroClient.ts

```diff
@@ -531,10 +545,9 @@ export class VeleroClient {
       if (!_.isObject(store.s3Compatible)) {
         throw new ReplicatedError("s3Compatible store configuration is required");
       }
-      backupStorageLocation.spec.config = {
-        region: store.s3Compatible!.region,
-        endpoint: store.s3Compatible!.endpoint,
-      };
+      backupStorageLocation.spec.config.region = store.s3Compatible!.region;
+      backupStorageLocation.spec.config.s3Url = store.s3Compatible!.endpoint;
```

Comment:
This was a bug. The BackupStorageLocation config expects a field named "s3Url" and our SnapshotStore type uses a field named "endpoint".